internal/brokers/manager.go

```diff
@@ -59,15 +59,20 @@ func NewManager(ctx context.Context, configuredBrokers []string, args ...Option)
 		f(&opts)
 	}
 
+	brokersConfPath := filepath.Join(opts.rootDir, opts.brokerCfgDir)
+	if example, err := useExampleBrokers(); example != "" {
```

Comment:
I would name `example` a little bit different, like `brokersConfPathWithExample` and reverse the logic: first, checking for nil, and in the else if, doing the assigmnent if not empty.